src/JavascriptTestTrait.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\commerce;
```

Comment:
Perhaps this should live under Drupal\Tests\commerce\FunctionalJavascript since it will only be used by other modules in that kind of namespace? @bojanz what are your thoughts? Being over picky? I know I had some test base items in just `Drupal\Tests\<module>`. But since this trait is only to be used for functional Javascript tests, maybe we should put it there.